sample/refresh_token_sample.py

```diff
@@ -48,15 +48,15 @@ def turn_on_logging():
     RESOURCE, 
     sample_parameters['username'],
     sample_parameters['password'],
-    sample_parameters['clientid'])
+    sample_parameters['clientId'])
```

Comment:
Changing the code behavior in these 2 places is technically a breaking change. In fact, I know some other developers who worked on this repo all changed their sample config files, rather than change the code. So we better not break their testing environment. How about we change  your [example line 23](https://github.com/AzureAD/azure-activedirectory-library-for-python/pull/45/files#diff-a541bf130e65be63781912f07c299284L23) to match your previous implementation `clientid`?